### DISCUSSION TITLE
Soybean Leaf.RelativeArea starts at 0.667 instead of 0.0

### DISCUSSION THREAD

**Discussion by BrianCollinss** [APSIMInitiative/ApsimX#8903] [Q&A]
(I thought [this](https://github.com/APSIMInitiative/ApsimX/issues/8663) might receive a response here)

In Soybean, the **Leaf.DMDemands.Structural.DeltaLAIPotential.Vegetative.Delta.RelativeArea.BeforeEndPod.RelativeArea** starts at 0.667. Does this mean leaf area = 0.667 from the beginning of simulations, i.e. even without any node on the main stem? Should there be one more line in the table (0,0)?

![image](https://github.com/APSIMInitiative/ApsimX/assets/39573622/bbdc8e81-c8ec-443c-b85b-eb883846b362)

**Comment by hol353** [comment]
> Does this mean leaf area = 0.667 from the beginning of simulations

Yes. I don't know the soybean model but I suspect it doesn't matter though. I suspect the code won't use this function when node number < 2.

**Comment by BrianCollinss** [comment]
I am unsure why you believe it does not use this function when node number < 2. 

![image](https://github.com/APSIMInitiative/ApsimX/assets/39573622/06ef34ff-b057-4efb-927d-dca93502b7d1)

**Comment by BrianCollinss** [comment]
https://github.com/APSIMInitiative/ApsimX/pull/8918#issuecomment-2106555829
